app/src/main/res/values/strings.xml

```diff
@@ -707,6 +707,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_generalFee_title">Does it cost a fee to enter %s?</string>
     <string name="quest_laundrySelfService_title">Is this laundry a self service laundry?</string>
     <string name="quest_handrail_title">Do these steps have a handrail?</string>
+    <string name="quest_address_street_title">What street does address %s belong to?</string>
```

Comment:
I changed this to `address %s` rather than `house number %s` because `addr:housenumber` is frequently used on buildings (and landuses) that are not houses. I would prefer to keep it this way, as house number is not always correct.

Reply:
ok